src/cli/test_command.zig

```diff
@@ -578,6 +579,26 @@ pub const TestCommand = struct {
         fail_on_low_coverage: bool = false,
     };
 
+    pub fn doesAbsolutePathMatchFilter(filter_names: []const []const u8, name_: string) bool {
+        if (filter_names.len == 0) return true;
+
+        for (filter_names) |filter_name| {
+            if (strings.startsWith(name_, filter_name)) return true;
```

Comment:
`hasPrefix` should be used here, not `startsWith`. `startsWith` includes empty strings.

```suggestion
            if (strings.hasPrefix(name_, filter_name)) return true;
```